charts/spark-operator-chart/templates/webhook-cleanup-job.yaml

```diff
@@ -12,6 +12,9 @@ spec:
     metadata:
       name: {{ include "spark-operator.fullname" . }}-webhook-cleanup
       {{- if .Values.istio.enabled }}
+      annotations:
+        "sidecar.istio.io/inject": "true"
```

Comment:
Why not `"sidecar.istio.io/inject": {{ .Values.istio.enabled | quote }}`

Reply:
I was applying the same style that already existed https://github.com/GoogleCloudPlatform/spark-on-k8s-operator/blob/2919f521ccde71fc54b8d72ba0474c5872c7d4aa/charts/spark-operator-chart/templates/webhook-cleanup-job.yaml#L16

Reply:
https://github.com/GoogleCloudPlatform/spark-on-k8s-operator/pull/1439/commits/8c0c27cca462a4b86a67c62e8e2c6bad745f2f04